packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -359,7 +361,12 @@ const DetailsListInner: React.ComponentType<IDetailsListInnerProps> = (
   const finalOnRenderDetailsGroupHeader = React.useMemo(() => {
     return onRenderDetailsGroupHeader
       ? (groupHeaderProps: IGroupDividerProps, defaultRender?: IRenderFunction<IGroupDividerProps>) => {
-          const { ariaPosInSet, ariaSetSize } = groupHeaderProps;
+          const { groupIndex } = groupHeaderProps;
+          const groupKey: string | undefined = groups && groupIndex !== undefined ? groups[groupIndex].key : undefined;
```

Comment:
Hello,

my app throws an error here:

TypeError: Cannot read property 'key' of undefined
(anonymous function)
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/components/DetailsList/DetailsList.base.tsx:365
  362 | return onRenderDetailsGroupHeader
  363 |   ? (groupHeaderProps: IGroupDividerProps, defaultRender?: IRenderFunction<IGroupDividerProps>) => {
  364 |       const { groupIndex } = groupHeaderProps;
> 365 |       const groupKey: string | undefined = groups && groupIndex !== undefined ? groups[groupIndex].key : undefined;
      | ^  366 |       const totalRowCount: number =
  367 |         groupKey !== undefined && groupedDetailsListIndexMap[groupKey]
  368 |           ? groupedDetailsListIndexMap[groupKey].totalRowCount
View compiled
GroupedListSection.render
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/components/GroupedList/GroupedListSection.tsx:246
  243 | return (
  244 |   <div
  245 |     ref={this._root}
> 246 |     {...(isDraggable && { draggable: true })}
      | ^  247 |     className={css(groupedListClassNames && groupedListClassNames.group, this._getDroppingClassName())}
  248 |     role="presentation"
  249 |   >
View compiled
▼ 19 stack frames were expanded.
finishClassComponent
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:17485
updateClassComponent
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:17435
beginWork
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:19073
HTMLUnknownElement.callCallback
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:3945
invokeGuardedCallbackDev
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:3994
invokeGuardedCallback
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:4056
beginWork$1
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:23964
performUnitOfWork
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:22776
workLoopSync
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:22707
renderRootSync
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:22670
performSyncWorkOnRoot
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:22293
(anonymous function)
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:11327
unstable_runWithPriority
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/scheduler/cjs/scheduler.development.js:468
runWithPriority$1
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:11276
flushSyncCallbackQueueImpl
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:11322
flushSyncCallbackQueue
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:11309
discreteUpdates$1
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:22420
discreteUpdates
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:3756
dispatchDiscreteEvent
C:/Dev/BRZ365.Viewer3D/BRZ365.Viewer3D.WebApplication/ClientApp/node_modules/react-dom/cjs/react-dom.development.js:5889
▲ 19 stack frames were expanded.

I use groups with childrens